### DISCUSSION TITLE
可否添加输入弹幕时,视频自动暂停功能?

### DISCUSSION THREAD

**Discussion by lolisix** [the1812/Bilibili-Evolved#1925] [功能建议 / Ideas]
本老年人打字慢,如果我边打字,视频边播放的话,弹幕能够延迟至少10秒钟.
ps:这个想法应该没有人提出过吧.

**Comment by Coulomb-G** [comment]
先回车就可不暂停视频,并开始输入弹幕
先按空格,再回车就可暂停视频,并开始输入弹幕

**Reply by lolisix** [reply]
道理是这个道理,但是脑子总是忘记先空格.每次都是打了几个字以后,才想起来没暂停,还要现移动鼠标才能暂停.
